build/Makefile

```diff
@@ -431,8 +431,16 @@ ensure-windows-buildx:
 	-DOCKER_CLI_EXPERIMENTAL=enabled docker buildx create --name=$(BUILDX_WINDOWS_BUILDER) --use
 
 # Build the image for the gameserver sidecar and SDK binaries
-build-agones-sdk-image: $(ensure-build-image) build-agones-sdk-binary build-licenses build-required-src-dist
+ifeq ($(WITH_ARM64), 1)
+build-agones-sdk-image: $(ensure-build-image) build-agones-sdk-image-amd64 build-agones-sdk-image-arm64
+else
+build-agones-sdk-image: $(ensure-build-image) build-agones-sdk-image-amd64
+endif
+
+build-agones-sdk-image-amd64: $(ensure-build-image) build-agones-sdk-binary build-licenses build-required-src-dist
 	docker build $(agones_path)/cmd/sdk-server/ --tag=$(sidecar_linux_amd64_tag) $(DOCKER_BUILD_ARGS)
+build-agones-sdk-image-arm64: $(ensure-build-image) build-agones-sdk-binary build-licenses build-required-src-dist
+	docker build --platform linux/arm64 --build-arg ARCH=arm64 $(agones_path)/cmd/sdk-server/ --tag=$(sidecar_linux_amd64_tag) $(DOCKER_BUILD_ARGS)
```

Comment:
Should I change the sidecar_linux_amd64_tag, with sidecar_linux_tag ?
Using amd64 during arm64 step is confusing